packages/cms-cli/commands/logs.js

```diff
@@ -97,78 +116,96 @@ const tailLogs = async ({
-      const { config: configPath } = program;
-      const portalId = getPortalId(program);
-      const { latest, file, follow, compact } = program;
-      let logsResp;
-
-      setLogLevel(program);
-      logDebugInfo(program);
-      loadConfig(configPath);
-      checkAndWarnGitInclusion();
-      trackCommandUsage(
-        COMMAND_NAME,
-        {
-          latest,
-          file,
-        },
-        portalId
-      );
-
-      logger.debug(
-        `Getting ${
-          latest ? 'latest ' : ''
-        }logs for function with path: ${functionPath}`
-      );
-
-      const functionResp = await getFunctionByPath(
+const action = async (args, options) => {
+  loadAndValidateOptions(options);
+
+  const { functionPath } = args;
+  const { latest, file, follow, compact } = options;
+  let logsResp;
+  const portalId = getPortalId(options);
+
+  trackCommandUsage(COMMAND_NAME, { latest, file }, portalId);
+
+  logger.debug(
+    `Getting ${
+      latest ? 'latest ' : ''
+    }logs for function with path: ${functionPath}`
+  );
+
+  const functionResp = await getFunctionByPath(portalId, functionPath).catch(
+    e => {
+      logApiErrorInstance(e, {
+        functionPath,
         portalId,
-        functionPath
-      ).catch(e => {
-        logApiErrorInstance(e, {
-          functionPath,
-          portalId,
-        });
-        process.exit(1);
       });
+      process.exit(1);
+    }
+  );
+
+  logger.debug(`Retrieving logs for functionId: ${functionResp.id}`);
 
-      logger.debug(`Retrieving logs for functionId: ${functionResp.id}`);
-
-      if (follow) {
-        await tailLogs({
-          functionId: functionResp.id,
-          functionPath,
-          portalId,
-          portalName: program.portal,
-          compact,
-        });
-      } else if (latest) {
-        logsResp = await getLatestFunctionLog(portalId, functionResp.id);
-      } else {
-        logsResp = await getFunctionLogs(portalId, functionResp.id);
-      }
-
-      if (logsResp) {
-        return outputLogs(logsResp, {
-          compact,
-        });
-      }
+  if (follow) {
+    await tailLogs({
+      functionId: functionResp.id,
+      functionPath,
+      portalId,
+
+      Name: options.portal,
+      compact,
     });
+  } else if (latest) {
+    logsResp = await getLatestFunctionLog(portalId, functionResp.id);
+  } else {
+    logsResp = await getFunctionLogs(portalId, functionResp.id);
+  }
+
+  if (logsResp) {
+    return outputLogs(logsResp, {
+      compact,
+    });
+  }
+};
 
-  addPortalOptions(program);
-  addLoggerOptions(program);
-  addHelpUsageTracking(program, COMMAND_NAME);
+// Yargs Configuration
+const command = `${COMMAND_NAME} <function_path>`;
+const describe = DESCRIPTION;
+const builder = yargs => {
+  addConfigOptions(yargs, true);
+  addPortalOptions(yargs, true);
+  addLoggerOptions(yargs, true);
+  yargs.positional('path', {
+    describe: 'Path to serverless function',
+    type: 'string',
+  });
+  return yargs;
```

Comment:
It looks like the other options like `--latest`, `--compact`, etc… haven't been added to the yargs version of the command.

Reply:
Ah jeez.  Adding now